string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,12 @@
+/* exported firstChars */
+
+function firstChars(length, string) {
+  var newString = '';
+
+  for (var i = 0; i < length; i++) {
+    if (string[i] !== undefined) {
+      newString += string[i];
+    }
```

Comment:
Try to handle the case where the `string` is shorter than `length` with a "guard" before the loop even starts.